Makefile

```diff
@@ -121,3 +121,8 @@ watch_css:
 sync:
 	rsync -azru --delete --stats public/ ../prism-break-static/public/
 	(cd ../prism-break-static; git add -A; git commit -m 'regenerate'; git push)
+
+# Localizations
+
+localize_%:
+	lsc ./source/functions/find-missing-localizations.ls $*
```

Comment:
This either needs to me added to the PHONY list or reworked so that the target is a file path. It looks like it is intended to be PHONY.